doonut-app-external-api/src/main/kotlin/com/doonutmate/member/controller/MemberController.kt

```diff
@@ -23,8 +26,10 @@ class MemberController(
         @Authorization
         @Parameter(hidden = true)
         memberId: Long,
+        @RequestParam oauthTypeStrategy: OauthTypeStrategy,
```

Comment:
API를 사용하는 클라이언트는 내부 구현에서 전략 패턴을 사용하는지 드러낼 필요가 없을 것 같아요.  
내부 구현은 언제든 변경될 수 있으니 oauthType으로 이름을 변경하는게 좋을 것 같습니다!  

```suggestion
        @RequestParam oauthType: OauthType,
```